.github/workflows/gh-actions-release.yml

```diff
@@ -0,0 +1,79 @@
+name: 'GH Actions - Release'
+
+on:
+  release:
+    types:
+      - created
+      - edited
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # only build one Deb File
+          #- FROM:     'ubuntu:focal'
+          #  COMPILER: 'clang'
+          #  FLAGS:    '-DUSE_PYTHON_3=ON'
+          #- FROM:     'ubuntu:bionic'
+          #  COMPILER: 'clang'
+          #  FLAGS:    '-DUSE_PYTHON_3=ON'
+          #- FROM:     'ubuntu:xenial'
+          #  COMPILER: 'gcc'
+          #  FLAGS:    '-DUSE_PYTHON_3=ON'
+          #- FROM:     'debian:buster'
+          #  COMPILER: 'clang'
+          #  FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'debian:stretch'
+            COMPILER: 'gcc'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'opensuse/leap'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'fedora:33'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'fedora:32'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'centos:8'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        # We must fetch at least the immediate parents so that if this is
+        # a pull request then we can checkout the head.
+        fetch-depth: 2
+        submodules: false
+
+    # where in a pull request we want to move to the head of the PR and not be at the
+    # merge commit; here we want to be at the HEAD of the ref
+
+    - name: Extract tag name etc
+      shell: bash
+      run: |
+        echo "TAG_NAME=$(echo ${GITHUB_REF#refs/tags/} | sed 's/\//_/g')" >> $GITHUB_ENV
+    - name: Test tag name etc
+      run: |
+        echo "${TAG_NAME}"
+    - name: Run CI
+      env:
+        FROM:       ${{ matrix.FROM }}
+        COMPILER:   ${{ matrix.COMPILER }}
+        FLAGS:      ${{ matrix.FLAGS }}
+        MY_OS_NAME: linux
+        IS_RELEASE: 1
+      run: script/cibuild $FLAGS
+    - name: Upload the artifacts
+      uses: skx/github-action-publish-binaries@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        args: 'packages/Vega-Strike_v*.*'
```

Comment:
See if you can do something like this:

```yml
with:
  args: [ 'packages/*.rpm', 'packages/*.deb' ]
```

Reply:
The base filename would be `vsUTCS_v*.*` in this case. And that won't exclude the source ZIP, the way it does for the engine, because of the different CPack naming strategy.

Reply:
Unfortunately I don't think that'd work as according to https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepswithargs the `job.jobid.steps.with.args` doesn't support arrays of strings, just a single string. :disappointed:

Reply:
I propose we table the sub-selecting for the moment. We can fix that in another PR.

Reply:
OK

Reply:
Oh wait. Why not just repeat the entire "Upload artifacts" step, only with a different wildcard pattern? I.e.,

```yml
    - name: Upload the artifacts - .deb
      uses: skx/github-action-publish-binaries@master
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        args: 'packages/vsUTCS_v*.deb'
    - name: Upload the artifacts - .rpm
      uses: skx/github-action-publish-binaries@master
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        args: 'packages/vsUTCS_v*.rpm'
```

Reply:
If that doesn't work, or if you want to wait to try it in another PR, then that's fine, and I'll approve this PR.

Reply:
I'd rather do that in another PR where we can explicitly experiment with it. That way the work on that is focused rather than appending multiple commits to this one to try things and then fix when GH complains or it doesn't work as expected.

Reply:
It may also be best to try that on the Engine since the releases here are significantly larger.

Reply:
OK, no problem. Approved then.